src/views/group_monitor.ts

```diff
@@ -630,13 +662,13 @@ export class KNXGroupMonitor extends LitElement {
       const info = await getGroupMonitorInfo(this.hass);
       this._isProjectLoaded = info.project_loaded;
 
-      // Merge new telegrams with existing ones instead of replacing
+      // Merge new telegrams with existing ones
```

Comment:
```suggestion
      this._maxTelegramStorage = info.recent_telegrams.length + 1000
      // Merge new telegrams with existing ones
```
If we used something like this (dynamic `this._maxTelegramStorage` instead of `const MAX_TELEGRAM_STORAGE = 30000;`) users could tweak this option directly in the integration settings. 
For those using older / low ram devices they can keep the integration default of 1k, for those with higher demands they can set it to 25k and the monitor would respect that.

Reply:
We could, in future, also add the configured value to the WS response. Then it would work from day 1 of integration usage - when the user hasn't received 25k telegrams yet.

Reply:
Very nice!